lib/common/common_helper.rb

```diff
@@ -78,13 +80,39 @@ def add_trailing_slash(url)
   url =~ /\/$/ ? url : "#{url}/"
 end
 
-def missing_db_file?
+def missing_db_files?
   DbUpdater::FILES.each do |db_file|
     return true unless File.exist?(File.join(DATA_DIR, db_file))
   end
   false
 end
 
+# Find data.zip?
+def has_db_zip?
+  return File.exist?(DATA_FILE)? true : false
```

Comment:
can just do `return File.exist?(DATA_FILE)` :)

Reply:
So you can! Thanks!